README.md

```diff
@@ -286,11 +286,24 @@ DEBUG level logging enables logging of:
 
 **Note:** Debug level logging may expose sensitive data in the logs
 
+## Security
+This library does not maintain or manage authentication tokens. The consumer of the SDK is responsible for writing code to retrieve, store, and refresh authentication tokens.
```

Comment:
The SDK does provide retrieval, just not store and refresh.  Does that matter?

Reply:
Let me check with Yassir Elley and Austin Siford, who provided that text to me.